apollo/src/graphql/user/team.ts

```diff
@@ -36,12 +35,11 @@ builder.objectType(Team, {
                 return user;
             },
         }),
-        owner: t.field({
-            type: User,
+        owners: t.field({
```

Comment:
since I'm using "owner" as a policy tool for the "Team Admin" role, a team can now have multiple owners. Given this, I removed the `owner_id` column from the db table and am resolving based on the team's owner edges

Reply:
this actually needs another quick update. Incoming shortly-ish

Reply:
nvm, needs more work than I want to do here. Will save for a future PR building out team queries